packages/use-dataloader/src/dataloader.ts

```diff
@@ -12,6 +12,10 @@ export type DataLoaderConstructorArgs<T = unknown> = {
 }
 
 class DataLoader<T = unknown> {
+  public static started = 0
+
+  public static maxConcurrent = DEFAULT_MAX_CONCURRENT_REQUESTS
```

Comment:
Is there a reason for `started` & `maxConcurent` to be static?

This could lead to weird side effects, IMO

Reply:
Yeah this properties are static like this all instances of DataLoader can share the reference to the value and be all up-to-date. The solid principle is kept because of DataLoaderProvider should only manage dataloader instance and not the request queue. Moreover keeping our complexity of request launches in the class allow us to keep a simple hook and Provider.